PRIVACY.md

```diff
@@ -0,0 +1,111 @@
+# Privacy Policy
+Last updated: 2023-08-24
+
+Your privacy is a fundamental right JabRef e.V. respects and supports.
+By using JabRef and its related online services, you choose to share some of this information.
+In this Privacy Policy We explain how We collect, use, and share information about you, along with the choices you have.
+
+Please remember that no method of transmission over the Internet, or method of electronic storage is absolute secure.
+While We strive to use commercially acceptable means to protect your Personal Data, We cannot guarantee its absolute security.
+Also, whenever you communicate through the internet, your IP-Address will be transmitted and retained by other parties for technical reasons.
+
+The term 'Personal information' in this policy means any information that either directly identifies you or can be somehow linked to you. 'JabRef' is the software application, JabRef e.V. provides.
+
+## JabRef
+
+### Collecting information
+JabRef does not collect any personal information directly linked to you.
+However, on certain occasions the software application will send some information to the online services of JabRef e.V.:
+- On application start, JabRef will check for the latest version online (by default *enabled*).
+- Information about a journal you are citing when looking for more information about this journal, using our journal database (by default *enabled*).
+- A pdf document you automatically want to extract citation information from, using our GROBID service (by default *disabled*).
+- Anonymized statistical data on the use of the graphical user interface for internal analysis purposes (by default *disabled*).
+
+### Storing information
+JabRef does not store any personal information, except:
+- Your proxy username and password, if you decide to store them (by default *disabled*).
+- Any personal API key you use to access third party online services (by default *disabled*). 
+
+### Sharing information
+Certain operations you perform in JabRef may trigger requests to public third-party services such as Zotero, Crossref or the Library of Congress for metadata retrieval.
+These third parties may log besides your IP address and the search terms (e.g., DOI, ISBN or the current URL) according to their privacy policies.
```

Comment:
```suggestion
These third parties may log additional information besides your IP address and the search terms (e.g., DOI, ISBN or the current URL) according to their privacy policies.
```

Reply:
I would emphasize that we don't send any personal information to these services, except for what is strictly necessary to perform the task. In particular, only the ip Adress and the provided search terms are thansfered.